Library/Formula/unittest-cpp.rb

```diff
@@ -1,21 +1,44 @@
-require 'formula'
-
 class UnittestCpp < Formula
-  homepage 'http://unittest-cpp.sourceforge.net/'
-  url 'https://downloads.sourceforge.net/project/unittest-cpp/UnitTest++/1.4/unittest-cpp-1.4.zip'
-  sha1 'dad944159e2e135aea74039987490eaaee00f2ad'
+  homepage "https://github.com/unittest-cpp/unittest-cpp"
+
+  stable do
+    url "https://github.com/unittest-cpp/unittest-cpp/archive/v1.4.tar.gz"
+    sha1 "ec7bdbebeb6f4d7a069f1125f7b4f473198e491d"
+
+    # Clang failure fixed in the HEAD already
+    fails_with :clang do
+      build 600
+      cause "Failure in test: Expected 2 but was 0"
+    end
```

Comment:
Ugh. Nope, Sorry. The original patch wouldn't apply cleanly, and then I rebased it and it still wouldn't apply cleanly, so I settled for inreplacing the failed elements and even with the inreplace there the stable build still failed with Clang :anguished: